dockerfiles/startall.sh

```diff
@@ -46,10 +46,14 @@ while [ "$#" -gt 0 ]; do
          echo "    -d <docker-repo>   Docker repository to fetch images from"
          echo "    -l                 Build image label to use"
          echo "    -u                 Use "upgrade" rather than "install""
+         echo "    -t                 Generate templates instead of starting the system"
```

Comment:
cool, I had wanted to add that too.

There's a delay at the end of the script and a call to kubectl get pods.
Would be good if it avoided that if this option is on.
Should be separate PR really.

Reply:
nowsplit into a different pr